src/agat/agat_sp_filter_feature_from_kill_list/config.vsh.yaml

```diff
@@ -0,0 +1,105 @@
+name: agat_sp_filter_feature_from_kill_list
+namespace: agat
+description: |
+  The script aims to remove features based on a kill list. The default
+  behaviour is to look at the features's ID. If the feature has an ID
+  (case insensitive) listed among the kill list it will be removed. /!\
+  Removing a level1 or level2 feature will automatically remove all linked
+  subfeatures, and removing all children of a feature will automatically
+  remove this feature too.
```

Comment:
```suggestion
  Remove features based on a kill list. The default
  behaviour is to look at the features's ID. If the feature has an ID
  (case insensitive) listed among the kill list it will be removed. /!\
  Removing a level1 or level2 feature will automatically remove all linked
  subfeatures, and removing all children of a feature will automatically
  remove this feature too.
```